package.json

```diff
@@ -45,20 +46,21 @@
     "knexjs"
   ],
   "peerDependencies": {
+    "knex": ">=2.4.2",
     "seneca": ">=3",
-    "seneca-entity": ">=18",
-    "knex": ">=2.4.2"
+    "seneca-entity": ">=18"
   },
   "devDependencies": {
-    "pg": ">=8.8.0",
     "@hapi/code": "9.0.1",
     "@hapi/lab": "25.0.1",
     "async": "3.2.4",
     "lab-transform-typescript": "3.0.1",
+    "pg": ">=8.8.0",
```

Comment:
for more predicable builds, let's use fixed version numbers

Reply:
Got it